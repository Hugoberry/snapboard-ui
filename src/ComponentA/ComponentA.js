import React from 'react'

export default function ComponentA (props) {
  return <div>Component A</div>
}

ComponentA.displayName = 'ComponentA'
