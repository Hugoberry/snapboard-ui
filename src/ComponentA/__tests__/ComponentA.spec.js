import React from 'react'
import ComponentA from '../ComponentA'
import renderer from 'react-test-renderer'

test('Renders basic ComponentA', () => {
  const component = renderer.create(
    <ComponentA />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
