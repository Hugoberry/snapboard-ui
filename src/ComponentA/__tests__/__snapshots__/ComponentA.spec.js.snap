// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders basic ComponentA 1`] = `
<div>
  Component A
</div>
`;
